### PR TITLE
refactor(states): update states to be synced with template-formula

### PR DESCRIPTION
**Pull request by n-rodriguez**
### PR progress checklist (to be filled in by reviewers)

- [ ] Changes to documentation are appropriate (or tick if not required)
- [ ] Changes to tests are appropriate (or tick if not required)
- [ ] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [ ] `[feat]`     A new feature
- [ ] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [x] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [ ] `[docs]`     Documentation changes
- [ ] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Documentation checklist

- [ ] Updated the `README` (e.g. `Available states`).
- [ ] Updated `pillar.example`.

### Testing checklist

- [ ] Included in Kitchen (i.e. under `state_top`).
- [x] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [ ] Updated the relevant test pillar.

### CODE REVIEW

File: lynis/package/install.sls [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+lynis/package/install:
+  pkg.installed:
+    - pkgs:
+      - lynis
+      {% if lynis.repo.use_customers_repo and lynis.repo.install_plugins %}
+      - lynis-plugins
+      {% endif %}
+    {% if lynis.use_repo %}
+    - refresh_db: True
```
Comment (myii) [inline lynis/package/install.sls]: I know `refresh_db` has been deprecated for `pkgrepo.managed` (replaced by `refresh`) and that's probably the same for `pkg` -- the [documentation](https://docs.saltstack.com/en/latest/ref/states/all/salt.states.pkg.html#salt.states.pkg.installed) appears to favour `refresh` as well.

**Review by myii — COMMENTED** [review]
Possibly one change to be made but otherwise looks good!

File: lynis/package/install.sls (hunk 1 above)
Comment (n-rodriguez) [reply lynis/package/install.sls]: sure!

Comment (n-rodriguez) [reply lynis/package/install.sls]: done!

**Review by myii — APPROVED** [review]

**Comment by myii** [comment]
Merged, thanks @n-rodriguez.

**Pull request merged**